packages/amplify-category-predictions/index.js

```diff
@@ -0,0 +1,53 @@
+import { promptConsoleSupportedCategory } from './provider-utils/supportedPredictions';
+
+const predictionsConsole = require('./provider-utils/awscloudformation/index');
+const inquirer = require('inquirer');
+
+const category = 'predictions';
+
+async function console(context) {
+  const { amplify } = context;
+  const { amplifyMeta } = amplify.getProjectDetails();
+
+  return promptConsoleSupportedCategory()
+    .then(async (result) => {
+      result = result.category;
+      const predictionsResources = [];
+      Object.keys(amplifyMeta[category]).forEach((resourceName) => {
+        if (result.services.includes(amplifyMeta[category][resourceName].service) && result.types.includes(amplifyMeta[category][resourceName][result.type])) {
+          predictionsResources.push({ name: resourceName, value: { name: resourceName, service: amplifyMeta[category][resourceName].service } });
+        }
+      });
+      if (predictionsResources.length === 0) {
+        context.print.error(`No ${result.category} console supported resource found.`);
+        process.exit(0);
+        return;
+      }
+      let resourceObj = predictionsResources[0].value;
+      if (predictionsResources.length > 1) {
+        const resourceAnswer = await inquirer.prompt({
+          type: 'list',
+          name: 'resource',
+          messages: `Select an ${result.category} resource`,
```

Comment:
Wouldnt the `result.category` just be `result`? Seems like on line 14 that you're setting result.category to result